extensions/jupyter-adapter/src/JupyterKernel.ts

```diff
@@ -726,7 +726,12 @@ export class JupyterKernel extends EventEmitter implements vscode.Disposable {
 		const command = args.join(' ');
 
 		// Create environment.
-		const env = { POSITRON: '1', POSITRON_VERSION: positron.version };
+		const env = {
+			POSITRON: '1',
+			POSITRON_VERSION: positron.version,
+			POSITRON_LONG_VERSION: `${positron.version}+${positron.buildNumber}`,
+			POSITRON_MODE: vscode.env.uiKind === vscode.UIKind.Desktop ? 'desktop' : 'web',
```

Comment:
Here and for the terminals, I currently have "web" because that is what VS Code uses internally throughout to talk about this kind of build. RStudio uses "server" instead. Is it a problem to use a different string for Positron, to indicate this isn't running on a desktop? It is certainly easy to put a different string here, if consistency with what RStudio has done is more important than internal consistency with how the rest of VS Code / Positron refer to this.

Reply:
Looks like we _should_ have `.rs.api.versionInfo()` return "server" rather than "web":
https://github.com/search?q=.rs.api.versionInfo&type=code

Do we want the env vars to also say "server"? Or "web"?

Reply:
For Positron specific variables, and that we're calling it MODE, I think it is fine to use desktop and server.

Reply:
Hmm, if the MODE is about what kind of client is connecting, then desktop and web make sense. If the MODE were about where the extension host is running, then desktop and server would sound better.

For the legacy .rs.api.versionInfo() API, I think it's fine to translate a mode there to match old behavior.

Reply:
If we ever needed the literal value of vscode's UIKind in the terminal, we could separately expose that in a different var.